package.json

```
@@ -85,6 +85,7 @@
     "tdex:run-init": "bash scripts/tdex-run-init",
     "tdex:run-init-notls": "bash scripts/tdex-run-init-notls",
     "tdex:run-notls": "bash scripts/tdex-run-notls",
+    "tdex:run-notls-nomac": "bash scripts/tdex-run-notls-nomac",
```

maybe better this naming?
```suggestion
    "tdex:run-insecure": "bash scripts/tdex-run-insecure",
```

---

Yes but we also have `tdex:run-notls`

---

yeah insecure = no tls and no macaroons